osu.Game/Overlays/KeyBinding/KeyBindingRow.cs

```diff
@@ -216,8 +216,11 @@ protected override bool OnKeyDown(KeyDownEvent e)
                 }
             }
 
-            bindTarget.UpdateKeyCombination(KeyCombination.FromInputState(e.CurrentState));
-            if (!isModifier(e.Key)) finalise();
+            if (!isModifier(e.Key))
+            {
+                bindTarget.UpdateKeyCombination(KeyCombination.FromInputState(e.CurrentState));
```

Comment:
i think this line should still be outside so the UI updates?

Reply:
I use the shift key to pause the game in stable. So would it be best to just do nothing when the user is in a textbox?